.github/ISSUE_TEMPLATE/feature-request.md

```
@@ -20,3 +21,8 @@ A clear and concise description of approach to be followed.
 
 **Additional context**
 Add any other context or screenshots about the feature request here.
+
+**Potential internship candidates**
+
+Please read this if you are planning to apply for a Palisadoes Foundation internship 
+- https://github.com/PalisadoesFoundation/talawa/issues/359
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)